kodilla-testing/src/main/java/com/kodilla/testing/shape/ShapeCollector.java

```diff
@@ -0,0 +1,43 @@
+package com.kodilla.testing.shape;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ShapeCollector {
+
+    private List<Shape> list = new ArrayList<>();
+
+
+
+    public void addFigure(Shape shape)
+    {
+
+        list.add(shape);
+
+    }
+
+    public void removeFigure(Shape shape)
+    {
+
+        list.remove(list.indexOf(shape));
+    }
+
+    public Shape getFigure(int n)
+    {
+        return list.get(n);
+    }
+
+    public int getSizeOfTheList()
+    {
+        return list.size();
+    }
+
+    public Shape  showFigures()
+    {
+        Shape figures = null;
+        for (Shape figure: list) {
+            figures = figure;
+        }
+        return figures;
```

Comment:
ta metoda obecnie zwraca ostatni dodany Shape. To zupelnie nieintuicyjne. 
Mialoby sens gdyby ta metoda printowala list / zwracala liste / zwracala toString listy